src/gui/touchscreengui.cpp

```diff
@@ -1124,3 +1051,110 @@ void TouchScreenGUI::show()
 
 	setVisible(true);
 }
+
+v2s32 TouchScreenGUI::getPointerPos() {
+	if (m_draw_crosshair)
+		return v2s32(m_screensize.X / 2, m_screensize.Y / 2);
+	return m_pointer_pos[m_move_id];
+}
+
+void TouchScreenGUI::emitMouseEvent(EMOUSE_INPUT_EVENT type) {
```

Comment:
code style, also elsewhere
```suggestion
void TouchScreenGUI::emitMouseEvent(EMOUSE_INPUT_EVENT type)
{
```